lib/metro-transit.js

```diff
@@ -16,23 +16,21 @@ var MetroTransit = function () {
 MetroTransit.prototype = Object.create(AlexaSkill.prototype);
 MetroTransit.prototype.constructor = MetroTransit;
 
-MetroTransit.prototype.eventHandlers = {
-  onSessionStarted: function (sessionStartedRequest, session) {
-    console.log('MetroTransit onSessionStarted requestId: ' + sessionStartedRequest.requestId +
-        ', sessionId: ' + session.sessionId);
-    // any initialization logic goes here
-  },
+MetroTransit.prototype.eventHandlers.onSessionStarted = function (sessionStartedRequest, session) {
```

Comment:
Oh, I am way over thinking it. It is probably just because we didn't specify what `onIntent` was after we overrode the eventHandlers object. Let me give that a shot. It is probably better to override individual methods anyway (to avoid things like this happening), so we should merge this.